packages/ddp-server/stream_server.js

```
@@ -179,23 +179,17 @@ Object.assign(StreamServer.prototype, {
-        // Store arguments for use within the closure below
-        var args = arguments;
-
-        // TODO replace with url package
-        var url = Npm.require('url');
-
+      const newListener = function (request, ...rest) {
         // Rewrite /websocket and /websocket/ urls to /sockjs/websocket while
         // preserving query string.
-        var parsedUrl = url.parse(request.url);
+        const parsedUrl = new URL(request.url, 'http://dummy');
         if (parsedUrl.pathname === pathPrefix + '/websocket' ||
```

ESLint

[error] Insert `⏎··········`

(prettier/prettier)